olive/engine/engine.py

```diff
@@ -1019,6 +1026,8 @@ def _run_pass(
             parent_model_id=input_model_id,
             from_pass=pass_name,
             pass_run_config=pass_config,
+            start_time=run_start_time,
```

Comment:
Maybe we should save the start and end times in the cached run json too? 

Otherwise, for cached runs, we won't have times for them. Like the record at line 966

Reply:
Done.